packages/kbn-securitysolution-utils/src/path_validations/index.ts

```diff
@@ -28,7 +28,7 @@ export const enum OperatingSystem {
   WINDOWS = 'windows',
 }
 
-export type TrustedAppEntryTypes = 'match' | 'wildcard';
+export type EntryTypes = 'match' | 'wildcard' | 'match_any';
```

Comment:
I'm wondering if instead of doing this change and all other related changes would be easier to have just a type with:

```typescript
export type EntryTypes = TrustedAppEntryTypes & 'match_any';
```

Reply:
We should check with @ashokaditya here. What I don't know is - when we moved these types to this `@kbn` package, it may impact other functionality (ex. exceptions) that we might not be aware of.